src/hello-world.cpp

```diff
@@ -0,0 +1,3 @@
+#include <string>
```

Comment:
It's standard C/C++ practice to include the header file defining the API, i.e., `#include "hello-world.hpp"`, which also pulls in your public dependencies (`<string>`).